app/config/ApplicationConfig.scala

```diff
@@ -90,9 +90,16 @@ class ApplicationConfig @Inject()(
 
   val payeYear: Int = config.getInt("paye.year")
 
+  // Shutter Boolean and Message Urls
   val saShuttered: Boolean = config.getBoolean("shuttering.sa")
+  lazy val saAtsUnavailableTitleUrl = config.getString("sa.ats.service_unavailable.title")
```

Comment:
How come these have Url in the names? They are not Urls, so the name is misleading here